doc/playbook.rst

```diff
@@ -195,3 +207,73 @@ On a VM with the Science Pipelines installed, a new APDB schema can be created i
 .. code-block:: sh
 
     make_apdb.py -c isolation_level=READ_UNCOMMITTED -c db_url="postgresql://postgres@localhost/postgres"
+
+
+Middleware Worker VM
+====================
+
+The ``rubin-utility-middleware`` VM on Google Compute Engine is intended as a general-purpose environment for working with Butler repositories.
+It can work with both local repositories and ones based on Google Storage.
+
+Built-in support:
+
+* a complete install of the Science Pipelines in ``/software/lsst_stack/``
+* a running instance of ``cloud_sql_proxy`` mapping the ``butler-registry`` database to port 5432
+* global configuration pointing Butler ``s3://`` URIs to Google Storage buckets
+
+The user is responsible for:
+
+* running ``/software/lsst_stack/loadLSST.sh`` on login
+* database authentication (see `Databases`_, above)
+* `Google Storage Authentication`_
+
+
+Google Storage Authentication
+-----------------------------
+
+To access `Google Storage-Backed Repositories`_, you must first set up Boto authentication.
+If you don't have one, `create an HMAC key`_ (this is *not* the same as the token for running the `tester`_); the relevant service account is ``service-620570835826@gs-project-accounts.iam.gserviceaccount.com``.
```

Comment:
I was able to use successfully the `prompt-image-upload@prompt-proto.iam.gserviceaccount.com`--not sure if that matters

Reply:
I think it matters a little bit from a tidiness perspective; given that it's what we were given for GS access (e.g., through `gsutil`), I see `service-620570835826` as an account for humans, and `prompt-image-upload` as reserved for `upload.py`.